src/arrays.js

```diff
@@ -42,10 +44,20 @@ export const removeNthElement2 = (index, array) => {
   // your code here
 };
 
-export const elementsStartingWithAVowel = (strings) => {
-  // your code here
+const startsWithAVowel = (gfrd) => {
+  const vowels = ['a','e','i','o','u']
+  const firstCharacter = gfrd.charAt(0);
+  return vowels.includes(firstCharacter)
```

Comment:
I always have line breaks before return statements if there is other code in the block (but not directly after `{`), so it is clearly visible where the function is returned out of.